docs/pages/inboxes/create-client.md

```diff
@@ -0,0 +1,68 @@
+## Create or build a client
+
+:::warning[🚧 XMTP V3 Alpha]
+
+The XMTP V3 SDKs are in **alpha** status and ready for you to start experimenting with.
+However, we do **not** recommend using alpha software in production apps. Software in this status will change as we add features and iterate based on feedback.
+
+
+XMTP V3 includes breaking changes from V2, most of which will be mitigated by the SDKs. Guidance about the migration path from V2 to V3 will be delivered in forthcoming XMTP V3 release notes.
+
+Join the [XMTP Devs 💪 group chat](https://converse.xyz/group-invite/e-KZyw77m-7sjUmEk5lgu) in Converse to keep up with the latest updates.
+
+:::
+
+### Create an account SigningKey
```

Comment:
I think we need to align Signer and SigningKey usage

The functions are returning Signer, but then using a SigningKey in
`Client.createV3(SigningKey, {`
below